app/views/DisplayPrefsView.tsx

```diff
@@ -45,53 +45,53 @@ const DisplayPrefsView = (props: IDisplayPrefsView): JSX.Element => {
 		}
 	}, []);
 
-	const setSortPreference = async (param: Partial<IPreferences>) => {
+	const setSortPreference = (param: Partial<IPreferences>) => {
 		try {
 			dispatch(setPreference(param));
-			await RocketChat.saveSortPreference(param);
+			RocketChat.saveSortPreference(param);
 		} catch (e) {
 			log(e);
 		}
```

Comment:
I'm not sure it's worth `try/catch`ng it anymore.
I think nothing is going to throw.